Lib/test/test_pdb.py

```diff
@@ -20,8 +20,7 @@
 from test.support.pty_helper import run_pty, FakeInput
 from unittest.mock import patch
 
-# gh-114275: WASI fails to run asyncio tests, similar skip than test_asyncio.
-SKIP_ASYNCIO_TESTS = (not support.has_socket_support)
+SKIP_CORO_TESTS = False
```

Comment:
I left this in otherwise the diff is a bit noisy without whitespace hidden.

happy to remove

Reply:
I think it would be ok to remove this.
WDYM @sobolevn (as author of #114309)?

Reply:
Both are fine with me (I sightly prefer a smaller diff).

Reply:
ok I'll keep them for now, cc @Eclips4